src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,26 @@
 package calculator;
 
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.List;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        // 문자열을 입력 받아 저장할 수 있다.
+        System.out.println("덧셈할 문자열을 입력해주세요.");
+        String inputString = Console.readLine();
+
```

Comment:
```suggestion
        String input = Console.readLine();

```

해당 변수가 String이라는 것은 타입만 봐도 충분히 알 수가 있어요!
[좋은 코드를 위한 자바 변수명 네이밍](https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming/)
한 번 읽어보셔도 좋을 것 같아요.

Reply:
감사합니다 꼭 읽고 적용해보겠습니다!
리뷰를 받으면서 네이밍에 대한 고민이 많이 부족했다는 것을 느끼네요 👍 🫶